src/Components/Aspire.Azure.Messaging.ServiceBus/AzureMessagingServiceBusSettings.cs

```diff
@@ -44,14 +46,35 @@ public sealed class AzureMessagingServiceBusSettings : IConnectionStringSettings
 
     /// <summary>
     /// <para>Gets or sets a boolean value that indicates whether the OpenTelemetry tracing is enabled or not.</para>
-    /// <para>Disabled by default.</para>
     /// </summary>
     /// <remarks>
-    /// ActivitySource support in Azure SDK is experimental, the shape of Activities may change in the future without notice.
+    /// ServiceBus ActivitySource support in Azure SDK is experimental, the shape of Activities may change in the future without notice.
     /// It can be enabled by setting "Azure.Experimental.EnableActivitySource" <see cref="AppContext"/> switch to true.
     /// Or by setting "AZURE_EXPERIMENTAL_ENABLE_ACTIVITY_SOURCE" environment variable to "true".
     /// </remarks>
-    public bool Tracing { get; set; }
+    public bool Tracing
+    {
+        get { return _tracing ??= GetTracingDefaultValue(); }
+        set { _tracing = value; }
+    }
+
+    // default Tracing to true if the experimental switch is set
+    // TODO: remove this when ActivitySource support is no longer experimental
+    private static bool GetTracingDefaultValue()
+    {
+        if (AppContext.TryGetSwitch("Azure.Experimental.EnableActivitySource", out var enabled))
+        {
+            return enabled;
+        }
+
+        var envVar = Environment.GetEnvironmentVariable("AZURE_EXPERIMENTAL_ENABLE_ACTIVITY_SOURCE");
+        if (envVar is not null && (envVar.Equals("true", StringComparison.OrdinalIgnoreCase) || envVar.Equals("1")))
+        {
+            return true;
+        }
+
+        return false;
```

Comment:
Future nit - might be worth moving this to `AzureComponent` so we can reuse it for EventHubs too.

Reply:
(and CosmosDB until they remove feature flag)

Reply:
Will move when we need it in more places.